### PR TITLE
CC-1357: refactor logger.Infof calls for xread block tests

### PR DESCRIPTION
**Pull request by ryan-gang**
This pull request refactors the logger.Infof calls in the xread block tests to use formatted strings. This improves code readability and maintainability.

### CODE REVIEW

File: internal/test_helpers/fixtures/streams/pass [hunk 1]
```diff
@@ -4,7 +4,7 @@ Debug = true
 [33m[stage-44] [0m[94m$ ./spawn_redis_server.sh[0m
 [33m[stage-44] [0m[94m$ redis-cli xadd "pear" "0-1" "temperature 70"[0m
 [33m[stage-44] [0m[92mReceived response: ""0-1""[0m
-[33m[stage-44] [0m[94m$ redis-cli xread block '\x00' streams "pear 0-1"[0m
+[33m[stage-44] [0m[94m$ redis-cli xread block 0 streams "pear 0-1"[0m
```
Comment (rohitpaulk) [inline internal/test_helpers/fixtures/streams/pass:7]: @ryan-gang can you check and see if "pear 0-1" is supposed to be quoted? I don't think it is: 

Screenshot 2024-08-08 at 15 12 20

**Review by rohitpaulk — CHANGES_REQUESTED** [review]
Added note re: quoting

File: internal/test_helpers/fixtures/streams/pass (hunk 1 above)
Comment (ryan-gang) [reply internal/test_helpers/fixtures/streams/pass:7]: Yep, shouldn't be quoted as per docs. 
But, as there is spacing inside, we kept it like this. Should I remove this then ?
This works with redis either way.

Comment (rohitpaulk) [reply internal/test_helpers/fixtures/streams/pass:7]: Yep let's remove quoting - seems like redis might have some kind of special handling for this if it works with quotes? The arguments seem to be separate arguments (one for key, one for id)

**Comment by ryan-gang** [comment]
We use all unquoted params for `XREAD`, as per https://redis.io/docs/latest/commands/xread/.

**Comment by ryan-gang** [comment]
Merging this in @rohitpaulk.

**Pull request merged**